desktop_server/src/com/mygdx/game/server/network/EndTurnService.java

```diff
@@ -0,0 +1,77 @@
+package com.mygdx.game.server.network;
+
+import com.artemis.World;
+import com.badlogic.gdx.utils.Json;
+import com.mygdx.game.core.network.messages.ChangeTurnMessage;
+import com.mygdx.game.server.ecs.entityfactory.ComponentFactory;
+import com.mygdx.game.server.model.Client;
+import com.mygdx.game.server.model.GameRoom;
+import io.vertx.core.buffer.Buffer;
+import lombok.NonNull;
+import lombok.extern.java.Log;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.List;
+
+@Log
+@Singleton
+public class EndTurnService {
+
+  private final GameRoom gameRoom;
+  private String currentToken;
+
+  private final Json json = new Json();
+
+  @Inject
+  EndTurnService(
+      @NonNull GameRoom gameRoom
+  ) {
+    this.gameRoom = gameRoom;
+  }
+
+  public void init() {
+    this.currentToken = gameRoom.getClient(0).getPlayerToken();
+  }
+
+  public void nextTurn(@NonNull Client client) {
```

Comment:
ta część nadal jest dla mnie dziwna, w sensie takie szukanie klienta itd. jak można to zmienić - dodatkowo nie ma tego kodu, który przygotowywałby na ustalenie kolejności

Reply:
No bo to nie jest zadanie tego issue tzn. kolejność,

Reply:
eh